package.json

```diff
@@ -43,6 +43,14 @@
     "hubot": "./bin/hubot"
   },
   "scripts": {
-    "test": "./script/test"
+    "clean": "rm -rf lib/",
+    "preinstall": "script/dotenv",
```

Comment:
Do we know if using `pre<thing>` works for these? If npm forks to run these, I don't think the changed environment would get back to the npm process that then calls the `<thing>`. My impression is these are more for like if you needed to run `./configure` which would create some files and whatnot.